From 528478505afa6db3206f6fabee15b9a26e26cb58 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marcus=20M=C3=BCller?= <marcus@hostalia.de>
Date: Mon, 5 Mar 2018 14:34:17 +0100
Subject: [PATCH 101/112] runtime: py_feval now includes Python.h

feels like this was an oversight; part of Maitland Bottoms ABI
compatibility checker happiness patches:
https://sources.debian.org/data/main/g/gnuradio/3.7.11-10/debian/patches/make-acc-happy
---
 gnuradio-runtime/include/gnuradio/py_feval.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gnuradio-runtime/include/gnuradio/py_feval.h b/gnuradio-runtime/include/gnuradio/py_feval.h
index cef168c8f0..89491af0b4 100644
--- a/gnuradio-runtime/include/gnuradio/py_feval.h
+++ b/gnuradio-runtime/include/gnuradio/py_feval.h
@@ -23,6 +23,7 @@
 #ifndef INCLUDED_GR_PY_FEVAL_H
 #define INCLUDED_GR_PY_FEVAL_H
 
+#include <Python.h>
 #include <pmt/pmt.h>
 #include <gnuradio/feval.h>
 
-- 
2.11.0

